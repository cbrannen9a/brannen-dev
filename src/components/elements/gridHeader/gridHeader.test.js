import React from 'react';
import renderer from 'react-test-renderer';
import GridHeader from './gridHeader';

describe('GridHeader', () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(
                <GridHeader
                    title={'Test'}
                    link={'test'}
                    buttonColour={'link'}
                    buttonTitle={'test'}
                />
            )
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});
