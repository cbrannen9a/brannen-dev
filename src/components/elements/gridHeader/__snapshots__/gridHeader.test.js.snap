// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GridHeader renders correctly 1`] = `
<h1
  className="title grid-header"
>
  Test
  <a
    className="button is-link"
    href="/test/"
    style={
      Object {
        "marginLeft": "10px",
      }
    }
  >
    test
  </a>
</h1>
`;
